DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/ElementConversionToSpeckle.cs

```diff
@@ -0,0 +1,66 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.Common;
+using Objects.BuiltElements.Revit;
+using Speckle.Converters.RevitShared.Helpers;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+[NameAndRankValue(nameof(Element), 0)]
+public class ElementConversionToSpeckle : BaseConversionToSpeckle<DB.Element, RevitElement>
+{
+  private readonly DisplayValueExtractor _displayValueExtractor;
+
+  public ElementConversionToSpeckle(DisplayValueExtractor displayValueExtractor)
+  {
+    _displayValueExtractor = displayValueExtractor;
+  }
+
+  public override RevitElement RawConvert(Element target)
+  {
+    RevitElement speckleElement = new();
+
+    if (target.Document.GetElement(target.GetTypeId()) is FamilySymbol symbol)
+    {
+      speckleElement.family = symbol.FamilyName;
+      speckleElement.type = symbol.Name;
+    }
+    else
+    {
+      speckleElement.type = target.Name;
+    }
+    speckleElement.type = target.Name;
+
+    //var baseGeometry = LocationToSpeckle(target);
+    //if (baseGeometry is Geometry.Point point)
+    //{
+    //  speckleElement["basePoint"] = point;
+    //}
+    //else if (baseGeometry is Geometry.Line line)
+    //{
+    //  speckleElement["baseLine"] = line;
+    //}
+
+    speckleElement.category = target.Category.Name;
+
+    speckleElement.displayValue = _displayValueExtractor.GetDisplayValue(target);
+
+    //GetHostedElements(speckleElement, target, out notes);
```

Comment:
Commented out section

Reply:
yeah, a lot of existing functionality that I haven't re-implemented